test/splitter.js

```diff
@@ -0,0 +1,160 @@
+const truffleAssert = require('truffle-assertions');
+const Splitter = artifacts.require("./Splitter.sol");
+
+contract('Splitter', async accounts => {
+
+    let gasCost = async receipt => {
+        const gasUsed = receipt.receipt.gasUsed;
```

Comment:
If you need to extract the receipt from it, it means the variable `receipt` is poorly named: it is not a receipt. 
From line 43, I know this is a `txObj` or something like that.

Reply:
I am guilty of cut and paste there.